### DISCUSSION TITLE
How do I log into the Plex LXC to install Plex update script?

### DISCUSSION THREAD

**Discussion by Allistah** [tteck/Proxmox#1724] [Q&A]
I'm trying to figure out how to log into the Plex LXC so that I can use the Plex Updater script.  What's the username and password for it the LXC that is created using the Proxmox Plex Helper script?  Thank you!

**Comment by lslvxy** [comment]
use `pct enter ID` to login

**Reply by NelsonMinar** [reply]
AFAICT the installer script never sets a password for root or the plex user. `pct enter` or the web console are the only ways I could get a shell.

**Comment by tteck** [comment]
> What's the username and password for it the LXC that is created using the Proxmox Plex Helper script?

The username is `root`, and the password is the one you set during LXC creation.

**Comment by lucas-0liveira** [comment]
To log in to the Plex LXC created using the Proxmox Plex Helper script, typically, the default username and password are provided during the setup. However, if you don't have them or can't recall, it's advisable to check the official documentation or resources provided by the creator of the Proxmox Plex Helper script.
